rust/src/tracing/mod.rs

```diff
@@ -0,0 +1,7 @@
+mod conversion;
```

Comment:
`mod.rs` is the old style of module. Please rename this file to `tracing.rs` and move it up one level in the hierarchy. `rust/src/tracing.rs`. https://doc.rust-lang.org/rust-by-example/mod/split.html